hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/hive/HoodieCombineHiveInputFormat.java

```diff
@@ -401,13 +407,14 @@ private static PartitionDesc getPartitionFromPath(Map<Path, PartitionDesc> pathT
       throws IOException {
     Method method;
     try {
-      Class<?> hiveUtilsClass = Class.forName("org.apache.hadoop.hive.ql.io.HiveFileFormatUtils");
+      Class<?> hiveUtilsClass = Class.forName("org.apache.hudi.hadoop.utils.HudiHiveFileFormatUtils");
```

Comment:
Looks good to me, how about adding a shim class to handle differences between versions? like we do in `org.apache.hudi.hadoop.utils.shims`.